pybloqs/block/image.py

```diff
@@ -244,16 +237,23 @@ def __init__(self, contents, **kwargs):
                        It is also useful in case a styling parameter name clashes with a standard
                        block parameter.
         """
+        # Imports are local so we do not need to install Plotly when it is not used
+        from plotly.graph_objs import Figure as PlotlyFigure
```

Comment:
Importing both here and at the bottom of the file in the `try`/`except` seems a bit weird.

Could you move the `try`/`except` to the top of the file and set a constant which you use at the bottom of the file to decide whether to call `add_block_types`? Something like:

```python
try:
    from plotly.graph_objs import Figure as PlotlyFigure
    _PLOTLY_AVAILABLE = True
except ImportError:
    _PLOTLY_AVAILABLE = False

...

class PlotlyPlotBlock(Block):
    ...

...

if _PLOTLY_AVAILABLE:
    add_block_types(PlotlyFigure, PlotlyPlotBlock)
```